src/ecalc/libraries/libecalc/common/libecalc/core/consumers/legacy_consumer/consumer_function/results.py

```diff
@@ -41,6 +41,8 @@ class ConsumerFunctionResultBase(BaseModel):
 
     # New! to support fuel to power rate...for e.g. compressors emulating turbine
     power: Optional[NDArray[np.float64]]
+    requested_inlet_pressure: Optional[NDArray[np.float64]]
```

Comment:
this should not be here, but on the compressorresult, at least for now, as it should only be for compressors...right?

Reply:
Fixed